pulpcore/cli/deb/context.py

```diff
@@ -15,6 +16,77 @@
 _ = translation.gettext
 
 
+class PulpDebGenericContentContext(PulpContentContext):
+    ENTITY = "apt generic content"
+    ENTITIES = "apt generic contents"
+    HREF = "deb_apt_generic_content_href"
+    ID_PREFIX = "content_deb_generic_contents"
```

Comment:
I don't know the correct naming convention here. I had to at least for `HREF` make changes for the upload to work. But I couldn't figure out where to look for it.

Reply:
Changed the naming to be similar as the corresponding entity on the clients